src/BoilerPlate/App.cpp

```diff
@@ -2,9 +2,10 @@
 #include <iostream>
 #include <algorithm>
 
+
 // OpenGL includes
 #include <GL/glew.h>
-#include <SDL2/SDL_opengl.h>
+#include <SDL_opengl.h>
```

Comment:
Please move SDL related stuff to it's own include file. It will help when you have to add stuff. You won't have to revisit all of your source files.